pkg/kubectl/cmd/autoscale.go

```diff
@@ -162,7 +161,11 @@ func RunAutoscale(f cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []s
 			object = hpa.Object
 		}
 		if cmdutil.GetDryRunFlag(cmd) {
-			return f.PrintObject(cmd, mapper, object, out)
+			if len(cmdutil.GetFlagString(cmd, "output")) > 0 {
+				return f.PrintObject(cmd, mapper, object, out)
+			}
+			cmdutil.PrintSuccess(mapper, false, out, info.Mapping.Resource, info.Name, true, "autoscaled")
```

Comment:
@adohe, according to other command's dry-run flag, this command will print sth like:
"deployment "dep" autoscaled (dry run)"
don't you think the tail `dry run` message is not clear enough?